PendingReleaseNotes

```diff
@@ -43,3 +43,8 @@
   Alternatively, the warning can be silenced with::
 
     ceph config set global mon_warn_on_pool_pg_num_not_power_of_two false
+
+* The behaviour of the ``-o`` argument to the rados tool has been reverted to
+  its orignal behaviour of indicating an output file. This reverts it to a more
+  consisten behaviour when compared to other tools. Specifying obect size is now
```

Comment:
s/obect/object/

Reply:
s/consisten/consistent/

we might want to fix these typos in master first though.